by-data-hash/36c971b8d41fbd94aabca77dc75e826729ac98447b46f91e00796155dddb0d29.md

```diff
@@ -0,0 +1 @@
+43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb.md
```

Comment:
```suggestion
Save as [43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb](43400de165f0821abf63dcac299bbdf7fd73898675ee4ddb099b0a0d8db63bfb.md)
```

Reply:
Or simply duplicate the file, otherwise the link anchor with the error code does not work.